ddtrace/propagation/http.py

```diff
@@ -103,6 +101,8 @@ class _DatadogMultiHeader:
       - ``x-datadog-origin`` optional name of origin Datadog product which initiated the request
     """
 
+    _PROPAGATED_TAGS = frozenset(["_dd.p.dm", "_dd.propagation_error"])
```

Comment:
```suggestion
    _PROPAGATED_TAGS = frozenset(["_dd.p.dm"])
```

`_dd.propagation_error` is only added to the span meta, it doesn't get propagated (... right?)